### PR TITLE
feat(connector): postmark email connector

### PR DESCRIPTION
**Pull request by srs**
## Summary

Implemented a Postmark email connector that uses Postmark API rather than SMTP. This provides the ability to use templates stored in Postmark instead of the templates stored in JSON. 

This is implemented by looking at `connector-sendgrid-email` and implemented it as my understanding of best practices and convensions. 

## Testing

Have a simple uint-test that only checks configuration is valid (as also `connector-sendgrid-email` has). Else I did manual testing by connecting it in Logto UI and sending a test email trough the UI. Have also tested it in real-world scenarios by connecting it to an email provider.

## Checklist

- [x] `.changeset`
- [x] unit tests
- [ ] integration tests
- [ ] necessary TSDoc comments

### CODE REVIEW

**Comment by srs** [comment]
This solves #5885.

**Comment by simeng-li** [comment]
@wangsijie @darcyYe can you guys take a look thx.

File: packages/connectors/connector-postmark/CHANGELOG.md
Comment (wangsijie) [inline packages/connectors/connector-postmark/CHANGELOG.md:1]: This file will be auto generated.

**Review by wangsijie — COMMENTED** [review]
The code looks good, could you add some basic test cases?

**Comment by wangsijie** [comment]
Also, it would be nice if you can make the commits signed.

**Comment by srs** [comment]
I will add some basic tests and do signed commits.

**Comment by charIeszhao** [comment]
Hey just ask our designer to create a logo for you.
logo

**Comment by charIeszhao** [comment]
The pnpm lockfile seems to be corrupted. Please try to resolve this issue in order to pass the CI checks.

Usually this can be resolved by running `pnpm i`

**Comment by ErickLuis00** [comment]
@srs

**Comment by srs** [comment]
Sorry for this. Will be right on it after my vacation that ends in 4 days :)

**Review by darcyYe — APPROVED** [review]

**Comment by darcyYe** [comment]
We can replace the logo before merging.

**Pull request merged**